.changelog/17309.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+tls: Add the ability to customize the details of the CA
```

Comment:
We tend to group them by process, so this would be `cli`:
```suggestion
cli: Add the ability to customize the details of the CA when running `nomad tls ca create`
```